Makefile

```diff
@@ -10,12 +10,14 @@ pgbouncer_SOURCES = \
 	src/hba.c \
 	src/janitor.c \
 	src/loader.c \
+	src/messages.c \
 	src/main.c \
 	src/objects.c \
 	src/pam.c \
 	src/pktbuf.c \
 	src/pooler.c \
 	src/proto.c \
+	src/prepared_statement.c \
```

Comment:
Look at the current name pattern. There is no underline separating words. Short name (even with abbreviation) is used. If I want to name it, I would use prepare.c (same as Postgres file) or prepstmt.c.

```
$ ls src/
admin.c   dnslookup.c  loader.c    objects.c  pooler.c              sbuf.c    stats.c     util.c
client.c  hba.c        main.c      pam.c      prepared_statement.c  scram.c   system.c    varcache.c
common    janitor.c    messages.c  pktbuf.c   proto.c               server.c  takeover.c
```

Reply:
`prepare.c` sounds good to me